Backend/DataAccessLayer/Migrations/201903031732527_create-table-point.cs

```diff
@@ -0,0 +1,31 @@
+namespace DataAccessLayer.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class createtablepoint : DbMigration
+    {
+        public override void Up()
+        {
+            CreateTable(
+                "dbo.Points",
+                c => new
+                    {
+                        Id = c.Guid(nullable: false),
+                        Name = c.String(nullable: false),
+                        Description = c.String(),

```

Comment:
This should probably be a text field (something with support for longer length entry)

Reply:
Not sure how to do this. It doesn't appear, from what I've very quickly looked at, that strings are given a capped length that would impede functionality here.

Reply:
Hmm, I looked online and I have to say I agree - can't find a limit for string length. I left this comment since most databases have a limit on the size of a string field - tends to be you want to move to a TEXT field if you're going to store large length text.